client/src/components/ShowDetailViewModal.js

```diff
@@ -25,26 +36,33 @@ const Container = styled.div`
   position: absolute;
 `;
 
-function ShowDetailViewModal({ showDetails, toggleModal }) {
+function ShowDetailViewModal({ showDetails, toggleModal, isShowing }) {
   return (
-    <Container>
-      <CloseButton onClick={() => toggleModal()} />
-      <Poster showPoster={showDetails.poster} />
-      <TitleSection
-        showRating={showDetails.rating}
-        showTitle={showDetails.title}
-      />
-      <OverviewTextarea showOverview={showDetails.overview} />
-      <GenreList genres={showDetails.genres} />
-      <ActorList actors={showDetails.actors} />
-      <WatchlistButtons />
-    </Container>
+    <>
+      {isShowing ? (
```

Comment:
I don't see the need for this condition, since it's already conditional in the `client/src/components/PopularShows.js`

Reply:
it's not conditional though? at least not in the return function:
or what do you mean?

```javascript
return (
    <>
      <ShowDetailViewModal
        isShowing={isShowing}
        toggleModal={toggleModal}
        showDetails={selectedItem}
      />
```